pkg/agent/flowexporter/flowrecords/flowrecords.go

```diff
@@ -0,0 +1,79 @@
+package flowrecords
```

Comment:
We add "_" to connect words in file names, so this file should be flow_records.go or just records.go.

Reply:
Sure. Thanks for letting me know the convention.